app/file.go

```diff
@@ -1128,6 +1155,9 @@ func (a *App) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
 		}
 	}
 
+	// update mini preview if needed, done in the background to avoid delaying the response
+	// will save fileinfo with the preview added for future use
+	a.generateMiniPreview(fileInfo)
```

Comment:
Also, does this need to be called anywhere else like `GetFileInfos` or `GetFileInfosForPost`?

Reply:
correct. will add